src/components/DisplayPanel.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Fixing linting issues does not involve ignoring the problem, you actually need to make your code compliant with the code, unless it is expressed otherwise from the docs. Please do tend to that or give a thorough, convincing explanation on why you choose to ignore the rule.